package.json

```diff
@@ -10,16 +10,15 @@
     "preview": "vite preview"
   },
   "dependencies": {
-    "@faker-js/faker": "^9.0.0",
-    "bootstrap": "^5.3.3",
-    "husky": "^9.1.6",
```

Comment:
You can remove this if you want, there are some useful hooks you can add with this.  Like format on commit so you don't have to worry about fixing eslint stuff

Reply:
You have a husky hook in yoru package.json. Probably shouldn't remove this

Reply:
removed when fightring with eslint action
readded